app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/checkout/_progress-bar.less

```diff
@@ -195,4 +196,27 @@
             }
         }
     }
+
+//
+// Mobile max width 600px
+//
```

Comment:
should target for mobile device

Reply:
It is not duplicated. Some kind of thing like margin and width is only changing based on screen size.

Reply:
without wrap between media mixins your code styles probably duplicated in both final output files styles-m.css and styles-l.css
That's why code base use mixins for target mobile or desktop

Reply:
If you want apply for mobile move code between ```less & when (@media-common = true) {}```